src/System.Security.Cryptography.Algorithms/tests/DefaultRSAProvider.cs

```diff
@@ -17,6 +21,27 @@ public RSA Create(int keySize)
             rsa.KeySize = keySize;
             return rsa;
         }
+
+        public bool Supports384PrivateKey
+        {
+            get
+            {
+                if (!_supports384PrivateKey.HasValue)
+                {
+                    bool hasSupport = true;
+
+                    // For Windows 7 (Microsoft Windows 6.1) this is false for RSACng.
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                    {
+                        hasSupport = !RuntimeInformation.OSDescription.Contains("Windows 6.1");
+                    }
+
+                    _supports384PrivateKey = hasSupport;
```

Comment:
It could, but I didn't want to end up with some sort of random false positive, given that this library is all platforms.